src/main/webapp/jsp/adminHome.jsp

```diff
@@ -1,81 +1,17 @@
 <%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
-
-<%@ page import="teammates.common.util.Const" %>
-<%@ page import="teammates.ui.controller.AdminHomePageData"%>
-<%@ page import="static teammates.ui.controller.PageData.sanitizeForHtml" %>
-
-<%
-    AdminHomePageData data = (AdminHomePageData)request.getAttribute("data");
-%>
-<!DOCTYPE html>
-<html>
-<head>
-    <title>TEAMMATES - Administrator</title>
-
-    <link rel="shortcut icon" href="/favicon.png" />
-
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <link type="text/css" href="/bootstrap/css/bootstrap.min.css" rel="stylesheet"/>
-    <link type="text/css" href="/bootstrap/css/bootstrap-theme.min.css" rel="stylesheet"/>
-    <link type="text/css" href="/stylesheets/teammatesCommon.css" rel="stylesheet"/>
-
-    <!--[if lt IE 9]>
-        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
-        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
-    <![endif]-->
-
-    <script type="text/javascript" src="/js/googleAnalytics.js"></script>
-    <script type="text/javascript" src="<%= data.getjQueryFilePath() %>"></script>
-    <script type="text/javascript" src="<%= data.getjQueryUiFilePath() %>"></script>
-    <script type="text/javascript" src="/js/common.js"></script>
-    <script type="text/javascript" src="/bootstrap/js/bootstrap.min.js"></script>
-
-    <jsp:include page="../enableJS.jsp"></jsp:include>
-
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/admin" prefix="ta" %>
+<%@ taglib tagdir="/WEB-INF/tags/admin/home" prefix="adminHome" %>
+
+<c:set var="jsIncludes">
+    <script type="text/javascript" src="${data.jQueryFilePath}"></script>
+    <script type="text/javascript" src="${data.jQueryUiFilePath}"></script>
```

Comment:
Sorry for didn't notice this sooner. I think these two lines about jQuery are included in the `page.tag` so there is no need to put them here.